configuration/exodus-pipeline.yaml

```diff
@@ -48,6 +48,22 @@ Parameters:
       - false
     Default: false
     Description: Determines whether to create CloudTrail resources
+  codebuildrole:
+    Type: String
+    Default: None
+    Description: Code Build Role
```

Comment:
I agree with @rohanpm that these descriptions aren't currently helpful. They're not required so I suggests either removing them entirely or expanding on the description (see above parameter descriptions for reference).

Reply:
@dichen16 this is also missing a Lambda function role, which should be passed to the CloudFormation template in the "Deploy" stage of the pipeline. The exodus-lambda-deploy.yaml template will need to be updated as well to make use of the static Lambda role.